File: code/3f119.pas
program Bai119;
uses math;
var a, b, c, d:integer;
s: real;
begin
readln(a, b, c);
d := max(a, max(b,c));
s := (a*b*c) /(2*b);
 writeln(s:5:2);
end.

File: code/3f119.cpp
#include <iostream>
using namespace std;

int main() 
{
    int a,b,c,d,s;
    cin>>a>>b>>c;
    d=max(a,max(b,c));
    s=a*b*c/(2*d);
    cout<<s;
    return 0;
}